utils/timestamp_mixin.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class TimestampMixin(models.Model):
+    """
+    Mixin to add created_at and updated_at fields to models
+    """
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider adding type hints to fields</b></div>
<div id="fix">

Consider adding type hints to the `created_at` and `updated_at` fields in the `TimestampMixin` class for better code maintainability and IDE support.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -7,8 +7,8 @@
     Mixin to add created_at and updated_at fields to models
     """
     created_at: models.DateTimeField = models.DateTimeField(auto_now_add=True)
     updated_at: models.DateTimeField = models.DateTimeField(auto_now=True)
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/sleep-404/ExamEvoke/pull/8#issuecomment-2499580485>#1f7607</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged